transcendence-app/src/user/user.service.ts

```diff
@@ -14,6 +14,20 @@ export class UserService {
   retrieveUserWithId(id: string) {
     return this.users.find((user) => user.id === id) || null;
   }
+  retrieveUsers() {
+    return this.users;
+  }
+  retrieveUsernames(take: number, skip: number, sort: boolean): string[] {
+    const ret: string[] = [];
+    if (take < 0 || skip < 0 || skip >= this.users.length)
+      throw new BadRequestException('Provided values incorrect');
```

Comment:
Despite we are very used to not use curly braces in 42 in order to save function lines (:sweat: ) it is an industry standard to always surround functions with them because improves legibility and clarifies statements scope.
```suggestion
    if (take < 0 || skip < 0 || skip >= this.users.length) {
      throw new BadRequestException('Provided values incorrect');
    }
```

Reply:
Perfect. I take note for future commits